tests/actions/scroll.test.js

```diff
@@ -81,7 +81,8 @@ describe('Interactor actions - scroll', () => {
       expect(scrollCount).toBe(10);
     });
 
-    it('eventually throws an error when scrolling a non-scrollable element', async () => {
+    // CSS layout is not supported in jsdom; there is no concept of overflow and everything is scrollable
+    it.skip.jsdom('eventually throws an error when scrolling a non-scrollable element', async () => {
```

Comment:
Nice way of skipping tests for jsdom. I saw the iterations through this and this is the nicest 👍